modules/gitops-release-notes-1-15-0.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-15-0.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-15-0_{context}"]
+= Release notes for {gitops-title} 1.15.0
+
+{gitops-title} 1.15.0 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-15.0_{context}"]
+== Errata updates
+
+[id="RHEA-XXXX:NNNN-gitops-1-15-0-security-update-advisory_{context}"]
+=== RHEA-XXXX:NNNN - {gitops-title} 1.15.0 security update advisory
+
+Issued: XXXX-YY-NN
+
+The list of security fixes that are included in this release are documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHEA-XXXX:NNNN[RHEA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="breaking-change-1-15-0_{context}"]
+== Breaking change
+
+[id="removal-of-application-manager-cli-kam_{context}"]
+=== Removal of {gitops-title} Application Manager CLI, `kam`
```

Comment:
```suggestion
=== Removal of {gitops-title} Application Manager CLI, kam
```
From the [OpenShift doc guidelines](Do not use backticks or other markup in assembly headings.): "Do not use backticks or other markup in assembly headings."

Reply:
Thanks for reminding me about this. Fixed.